launcher/game/preferences.rpy

```diff
@@ -334,19 +332,19 @@ screen preferences():
 
                             add HALF_SPACER
 
-                            textbutton _("Orphan translations"):
+                            textbutton _("Check for orphan/obsolete translations"):
                                 style "l_checkbox"
-                                action ToggleSetMembership(persistent.lint_options, "--orphan-tl")
-                            textbutton _("Parameters overriding builtin names"):
+                                action InvertSelected(ToggleSetMembership(persistent.lint_options, "--no-orphan-tl"))
+                            textbutton _("Check for parameters overriding builtin names"):
                                 style "l_checkbox"
                                 action ToggleSetMembership(persistent.lint_options, "--builtins-parameters")
-                            textbutton _("Word count and character count for speaking characters"):
+                            textbutton _("Print word and character counts for speaking characters"):
                                 style "l_checkbox"
                                 action ToggleSetMembership(persistent.lint_options, "--words-char-count")
 
                             add SPACER
 
-                            textbutton _("Check Script (Lint)") action Jump("lint")
+                            textbutton _("Check Script (Lint)") action Call("lint")
```

Comment:
I don't think this belongs here. This is the global options page, and it's a project specific action.